articles/azure/azs-how-create-service-principals.md

```diff
@@ -0,0 +1,97 @@
+---
+title: How to create a service principal for Azure AD 
+description: Give applications access to Azure Stack resources by creating service principals
+services: azure-stack
+author: David Woffendin
+toc_rootlink: Users
+toc_sub1: How To
+toc_sub2:
+toc_sub3:
+toc_sub4:
+toc_title: How to create a service principal for Azure AD 
+toc_fullpath: Users/How To/azs-how-create-service-principals.md
+toc_mdlink: azs-how-create-service-principals.md
+---
+
+# How to Give applications access to Azure Stack resources
+
+## Overview
+
+In Azure Stack you are able to give an application access to stack resources by creating a service principal that uses Azure Resource Manager. A service principal lets you delegate specific permissions using **role-based access control**. As a best practice, you should use service principals for your applications. Service principals are preferable to running an app using your own credentials for the following reasons:
+
+* You can assign permissions to the service principal that are different than your own account permissions. Typically, a service principal's permissions are restricted to exactly what the app needs to do.
+* You do not have to change the app's credentials if your role or responsibilities change.
+* You can use a certificate to automate authentication when running an unattended script.
+
+### Example
+
+You have a configuration management app that needs to inventory Azure resources using Azure Resource Manager. You can create a service principal and assign it to the Reader role. This role gives the app read-only access to Azure resources.
+
+## Getting started
+
+Use the steps in this article as a guide to:
+
+* Create a service principal for your app.
+* Register your app and create an authentication key.
+* Assign your app to a role.
+
+## Create a service principal for Azure AD
+
+If your Azure Stack uses Azure AD as the identity store, you can create a service principal using the same steps as in Azure, using the Azure portal. Please check that you have the require Azure AD permissions before trying to create a service principle.
+
+### Creating a service principal
+
+To create a service principal for your application:
+
+1. Sign in to your Azure Account through the Public Azure portal.
+2. Select **Azure Active Directory** > **App registrations** > **Add**.
```

Comment:
Should be **New application registration** instead of **Add**

Reply:
done